README.md

```diff
@@ -20,7 +20,59 @@ We believe that a common way to share design tokens will unlock efficiency oppor
 
 ## Roadmap
 
-The DTWG was founded in June 2019. By the end of 2019, our aim is to have a standard working across at least 3 design or prototyping tools.
+The DTWG was founded in June 2019. By the end of 2019, our aim is to have a basic implementation working across:
+
+- at least 3 design or prototyping tools
+- additional design token management tools such as Theo, Style Dictionary, Invision DSM, and Diez
+- various open source community-driven projects
+
+![timeline](https://user-images.githubusercontent.com/85783/61584784-6f791000-ab02-11e9-8dfd-896cc9d147ba.png)
+
+### June 2019
+
+- Creation 🎉 
+- [Editor’s draft: format](#1)
+- [Editor’s draft: theming](#2)
+
+### July 2019
+
+- [Define legal status](#8), affiliation with a standards body
+- Get commitment from vendors
+- Find editor(s) for the colors module
+ 
+### August 2019
+
+- Working draft: format
+- Working draft: theming
+- Editor’s draft: colors level 1
+
+### September 2019
+
+- Working draft: colors level 1
+- Collect feedback on the colors working draft
+
+### October 2019
+
+- Candidate Recommendation: colors level 1
+- Vendors start working on pre-releases the candidate recommendation
+- Open source tools (Theo, Style Dictionary, and maybe others like System UI…) build pre-releases to support the working draft
+
+### November 2019
+
+- Community / public testing
+- Recommendation: format (and maybe theming too, depending on how earlier discussions go)
+
+### December 2019
+
+- Recommendation: colors level 1
+
+### 2020
+
+Issue recommendations on additional aspects of design tokens:
+
+- Dimensions
+- Colors level 2 (such as gradients, etc)
+- Grid
```

Comment:
+ Motion (timing and easing)?
+ Groups / compound tokens (like borders & typography)?